docs/source/optimization_ov.mdx

```diff
@@ -62,6 +62,36 @@ tokenizer.save_pretrained(save_dir)
 
 The `quantize()` method applies post-training static quantization and export the resulting quantized model to the OpenVINO Intermediate Representation (IR). The resulting graph is represented with two files: an XML file describing the network topology and a binary file describing the weights. The resulting model can be run on any target Intel device.
 
+## LLM weight-only quantization
+
+You can optimize the performance of text-generation LLMs by quantizing weights to various precisions that provides different performance-accuracy trade-off.
+
+```python
+from optimum.intel import OVModelForCausalLM
+
+model = OVModelForCausalLM.from_pretrained(model_id, load_in_8bit=True)
+```
+
+> **NOTE:** `load_in_8bit` is enabled by default for the models larger than 1 billion parameters.
+
+For the 4-bit weight quantization we recommend using the NNCF API like below:
+```python
+from optimum.intel import OVModelForCausalLM
+import nncf
+
+model = OVModelForCausalLM.from_pretrained(model_id, load_in_8bit=False)
+model.model = nncf.compress_weights(
+        model.model,
+        mode=nncf.CompressWeightsMode.INT4_SYM,
```

Comment:
why not add a `load_in_4bit` argument ?

Reply:
we will add a bit later when finalize INT4 workflow in NNCF

Reply:
might make sense to wait for it no what do you think ?

Reply:
@echarlaix could we merge this now (after review) and then update for INT4 later with a new PR? The docs are incorrect now and that's causing some confusion, so it would be really good to have correct docs again.

Reply:
works for me